plugin/storage/cassandra/options.go

```diff
@@ -70,6 +72,16 @@ type namespaceConfig struct {
 	Enabled   bool
 }
 
+// rateLimitConfig defines common arguments to a rate limiter.
+type rateLimitConfig struct {
+	operationsPerSecond float64
+	maxBurst            float64
```

Comment:
Is this parameter required?

Reply:
Meant to delete this. Thanks for the reminder.